lmdeploy/serve/openai/api_server.py

```diff
@@ -40,6 +40,9 @@ class VariableInterface:
     session_id: int = 0
     api_keys: Optional[List[str]] = None
     request_hosts = []
+    # following are for registering to proxy server
+    proxy_url: Optional[str] = None
+    api_server_url: Optional[str] = None
```

Comment:
Is it necessary? Can we use `http://{server_ip}:{server_port}` instead

Reply:
It is a global variable. `VariableInterface.api_server_url = f'{http_or_https}://{server_name}:{server_port}'`